index.html

```diff
@@ -253,41 +282,41 @@ <h3>Languages</h3>
     </main>
 
     <footer class="footer">
-      <div class="contact-content">
-        <h2 class="contact-header">Contant Us</h2>
-        <p class="contact-paragraph">
-          If you have an application you are interested in developing, a feature
-          that you need built or a project that needs coding. I’d love to help
-          with it
-        </p>
-      </div>
-      <form action="https://formspree.io/f/xjvlllyy" method="POST">
-        <input
-          name="name"
-          id="user-name"
-          type="text"
-          placeholder="Name"
-          maxlength="30"
-          required
-        />
-
-        <input
-          name="email"
-          id="user-email"
-          type="email"
-          placeholder="name@gmail.com"
-          required
-        />
-
-        <textarea
-          name="message"
-          maxlength="500"
-          required
-          placeholder="Write your message here"
-        ></textarea>
-        <button type="submit" >Get in Touch</button>
-        <img class="footer-img" src="image/Shape.png" alt="footer-image" />
-      </form>
-    </footer>
+        <div class="contact-content">
+          <h2 class="contact-header">Contant Us</h2>
+          <p class="contact-paragraph">
+            If you have an application you are interested in developing, a feature
+            that you need built or a project that needs coding. I’d love to help
+            with it
+          </p>
+          <div class="footer-items">
+            <form action="https://formspree.io/f/xjvlllyy" method="POST">
+              <input
+                name="name"
+                id="user-name"
+                type="text"
+                placeholder="Name"
+                maxlength="30"
+                required
+              />
+  
+              <input
+                name="email"
+                id="user-email"
+                type="email"
+                placeholder="name@gmail.com"
+                required
+              /><textarea
+                name="message"
+                maxlength="500"
+                required
+                placeholder="Write your message here"
+              ></textarea>
+              <button type="submit">Get in Touch</button>
+            </form>
+            <img class="footer-img" src="image/Shape.png" alt="footer-image" />
+        </div>
+        </div>
+      </footer>
```

Comment:
- [ ] It would be better if you enlarge your contact form to fit your [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). THis will enhance the beauty of your design 😄. I know you can make it better 😊 
1. **Your design** 
![image](https://user-images.githubusercontent.com/80612925/171938875-abdfc9d3-e287-4518-98f3-85bc5de3fcf2.png)
2. **The Figma design**
![image](https://user-images.githubusercontent.com/80612925/171939199-4bb7875c-a849-46ea-83a1-3f8701e4e69e.png)